### ISSUE TITLE
cmake failed

### ISSUE THREAD

**Issue by yanyanho** [akosba/jsnark#24]
-- Checking for module 'libcrypto'
--   No package 'libcrypto' found
CMake Error at /usr/local/Cellar/cmake/3.12.1/share/cmake/Modules/FindPkgConfig.cmake:436 (message):
  A required package was not found

**Comment by HarryR** [comment]
Have you tried installing the `libcrypto` development packages for your distribution ?

I assume you're using OSX, you can install libcrypto via brew

**Comment by yanyanho** [comment]
Yes ,thank you. I install libcrypto.     I ran the tests successfully .  Is there an interface to get proof from libsnark?
